tuxemon/states/combat/combat.py

```diff
@@ -473,6 +473,9 @@ def transition_phase(self, phase: CombatPhase) -> None:
             self.players[0].set_party_status()
             self.end_combat()
 
+            for monster, player in product(self.active_monsters, self.active_players):
```

Comment:
Using product here is weird given that the next method has

`                    if player is self.players[0]:`

Reply:
I removed  **if player is self.players[0]:**, it wasn't necessary to the evolution.

Reply:
Can we use zip instead of product?